public/javascript/main.js

```diff
@@ -385,7 +385,21 @@ $(function(){
 
       $(this).data('chart', chart);
     }
-  })
+  });
+
+  $(".js-download-with-term").on("click", function(e){
+    var term = $(e.target).data("term");
+    var house = $(e.target).data("house");
+    if (term && house) {
+      // inject query vars into URL before the target anchor
+      e.preventDefault();
+      var href = $(e.target).attr('href');
+      if (href.indexOf('#') == -1) {
+        href = href + "#"; // add anchor tag at end of URL
```

Comment:
Comments that simply state in English what a fairly simple piece of code is doing are usually a red flag. Perhaps extract this to a suitably named helper function instead?